C++/2. Variables.md

```diff
@@ -2,59 +2,78 @@
 
 ## Start
 
-Variables are containers which hold a specific something. They are words which contain something in them, which you can call or take by just typing out the word. Variables in C++ are created by first typing out their type. Each variable has a type, and there are a few types of variables:
+Variables are containers which can hold values of a specific type. They are words which contain something in them, which you can call or take by just typing out the word. Variables in C++ are created by first typing out their type. Each variable has a type, and there are an infinite number of types that you can create. The language provides a few primitives types, which can be used to build larger, or more complicated types, here are a few:
```

Comment:
Hey are you sure there are an infinite amount of types of variables?

Reply:
I believe that each unique enum, class, struct, pointer, and/or reference are a unique type.
For example, `int` is a primitive type, whereas `int&` is usually something user-defined. What do you think?

Reply:
eh i don't know i guess i agree with you